demo/scripts/kind-cluster-config.yaml

```diff
@@ -1,7 +1,5 @@
 kind: Cluster
 apiVersion: kind.x-k8s.io/v1alpha4
-featureGates:
-  DynamicResourceAllocation: true
```

Comment:
If you expect this to work with kind clusters >= 1.32.0, then you probably want to keep this.